recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: dynare_preprocessor
+  version: 0.0.1
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/master/preprocessor-master.tar.gz
+  sha256: 25e1b0f30d49b24daf520223e1d1919e9a93dae32d6156f8269014ef2f5aa85e
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
```

Comment:
Meson, bison, flex and the compiler should be in build, the rest belongs into host.